rswag/rswag
# Renames rswag-ui.rb to rswag_ui.rb initialize file created by generator

## Problem
When you run the install generator by:
```ruby
rails g rswag:install
```
or just for `rswag-ui`:
```
rails g rswag:ui:install
```

It will create the files below:
```
spec/swagger_helper.rb
config/initializers/rswag-ui.rb
config/initializers/rswag_api.rb
```

## Solution
The proposal is to rename `rswag-ui.rb` to `rswag_ui.rb` because that's the preferred way to name files in Ruby ([reference to RuboCop](https://docs.rubocop.org/rubocop/cops_naming.html#namingfilename), [reference to Ruby Style Guide](https://rubystyle.guide/#snake-case-files)).

By the way, the same was done for `rswag-api` initialize file in [this commit](https://github.com/rswag/rswag/commit/eb4e6045c50c19d9352d9405b2841285d0141867).

### The changes I made are compatible with:

### Checklist

### Steps to Test or Reproduce
- Start new Rails project.
- Add to your project and install `rswag` gem.
- Run the install generator `rails g rswag:install`.
- Check `config/initializers` directory.
